apps/myaccount/src/components/shared/header.tsx

```diff
@@ -271,16 +277,19 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (
     /**
      * Resolves the organization label in the header.
      */
-    const resolveOrganizationLabel = (): ReactElement => {
-        const organization: string =
-            tenantName == "carbon.super"
-                ? commonConfig.header.organization.replace("{{productName}}", productName)
-                : tenantName;
+    const resolveOrganizationLabel =  (): ReactElement => {
+        let organizationName: string = tenantName;
+
+        if (tenantName === "carbon.super") {
```

Comment:
Shall we add a constant for this

Reply:
Fixed with [2d47cf8](https://github.com/wso2/identity-apps/pull/6466/commits/2d47cf85f055a223fc2874c0d2c09a5f5aa1f628)